dotcom-rendering/src/web/components/CaptionToggle.tsx

```diff
@@ -0,0 +1,73 @@
+import { css } from '@emotion/react';
+
+const captionToggleStyles = css`
+	#the-checkbox {
+		/* Never show the input */
+		display: none;
+	}
+	#the-caption {
+		/* Hide caption by default */
+		display: none;
+	}
+	#the-checkbox:checked + #the-caption {
+		/* Show the caption if the input is checked */
+		display: block;
+	}
+`;
+
+const toggleStyles = css`
+	position: absolute;
+	bottom: 5px;
+	right: 5px;
+	width: 32px;
+	height: 32px;
+	z-index: 1;
+	/* We're using rgba here for the opactiy */
+	background-color: rgba(18, 18, 18, 0.6);
+	border-radius: 50%;
+	border: none;
+	cursor: pointer;
+
+	svg {
+		top: 0;
+		bottom: 0;
+		right: 0;
+		left: 0;
+		margin: auto;
+		position: absolute;
+		fill: white;
+	}
+`;
+
+const alignTopRightStyles = css`
+	bottom: auto;
+	top: 5px;
+`;
+
+export const CaptionToggle = ({
```

Comment:
Is this part of the work to reorder the elements?

Reply:
Yes, as mentioned above this is because we now have a caption toggle on the video. As this is used in multiple olaces now, I thought it made sense to refactor it into its own component but can refactor if this isn't DCR convention.